cli/functionary/package.py

```diff
@@ -26,6 +28,10 @@ def generateYaml(output_dir: str, name: str, language: str):
         path.write_text(yaml.dump(metadata))
 
 
+def get_environment_id():
+    environmentId = os.environ.get('FUNCTIONARY_ENVIRONMENT')
```

Comment:
You changed the function name, but this needed to changed to.  Or you can just return the `os.environ.get` call directly and drop the variable.